deploy/airflow1/example_dags/single_task_dag.py

```diff
@@ -16,9 +16,12 @@
     dag=dag,
     config_overrides={
         "type": "Scala",
-        "sparkVersion": "3.0.0",
-        "mainApplicationFile": "local:///opt/spark/examples/jars/spark-examples_2.12-3.0.0.jar",
+        "sparkVersion": "3.2.0",
+        "interactive": False,
```

Comment:
Can you remove references to the `interactive` field everywhere?

Reply:
done